main_test.go

```diff
@@ -183,3 +183,33 @@ func testCreateConfFile(dirs []string, resolver, deployer string, t *testing.T,
 	}
 	return nil
 }
+
+func runNewCli(t *testing.T, args ...string) {
+	artifactoryGoCli := tests.NewJfrogCli(execMain, "jfrog rt", "")
```

Comment:
artifactory**Go**Cli --> rtCli
Also, can we have all build tools tests use this function, instead of them relying on the *artifactoryCli* variable, defined in *artifactory_test.go*? This will complete the tests separation we started. I guess we better implement this change as part of a different PR.